src/StrategyConvexStaker.sol

```diff
@@ -78,7 +78,7 @@ contract StrategyConvexStaker is
      */
     function _deployFunds(uint256 _amount) internal override {
         // the final true argument means we deposit + stake at the same time
-        booster.deposit(pid, _amount, true);
+        // booster.deposit(pid, _amount, true);
```

Comment:
should this be uncommented?

Reply:
the stableswap pool we deploy here isnt integrated into convex